src/Components/addItem.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { fb } from '../lib/firebase';
+import { FirestoreProvider } from 'react-firestore';
+
+class Item extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      itemname: '',
```

Comment:
I generally like to use camelCase for my JavaScript variables. This helps my brain process the name of the variable when I'm reading it. the capital `N` in `itemName` helps me read this variable as two words. This is purely a stylistic thing that isn't important, but I just wanted to share in case you were interested! 📝